tools/pot/openvino/tools/pot/api/samples/object_detection/object_detection_simple_api.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from openvino.tools.pot import AccurracyAwareQuantizationParameters, quantize_with_accuracy_control, \
+                                export
+from openvino.tools.pot.utils.logger import init_logger
+from openvino.tools.pot.api.samples.utils.argument_parser import get_common_argparser
+from openvino.tools.pot.api.samples.object_detection.metric import MAP
+from openvino.tools.pot.api.samples.object_detection.data_loader import COCOLoader
+
```

Comment:
5 lines of imports, can it be also simplified somehow?

Reply:
I am not sure what you mean here. If we merge everything into one file in the samples it can be simplified here but will look cumbersome on the sample level.